src/server.c

```diff
@@ -5283,6 +5283,17 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    /* check if DEBUG/MODULE command not allowed */
+    if ((c->cmd->proc == debugCommand && !allowProtectedAction(server.enable_debug_cmd, c)) ||
+        (c->cmd->proc == moduleCommand && !allowProtectedAction(server.enable_module_cmd, c))) {
+        rejectCommandFormat(c,"%s command not allowed, you may enable it by "
+                              "editing the %s option in the Redis configuration file,"
+                              "and then restarting the server.",
```

Comment:
If the config values are "local", a user could connect locally and execute these commands.